administration/serializers.py

```diff
@@ -29,15 +28,15 @@ class Meta(UserSerializer.Meta):
             "email",
             "phone_number",
             "is_active",
-            "is_staff"
+            "is_staff",
         )
         read_only_fields = ("phone_number", "email")
 
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = CustomUser
-        fields = ('email', 'name', 'surname')
+        fields = ("email", "name", "surname")
```

Comment:
I really dislike such approach, quotes reflecting context in a long run make work with code more efficient. But.. as you wish.

Reply:
It's not me, it's linter

Reply:
@BelousSofiya I know that it's linter. :)